teps/0091-trusted-resources.md

```diff
@@ -254,47 +278,52 @@ metadata:
   name: config-trusted-resources
   namespace: tekton-pipelines
 data:
-  onError: "stopAndFail"
+  verify: true
   cosign-pubkey-path: "/etc/signing-secrets/cosign.pub"
   kms-pubkey-path: "gcpkms://projects/<project>/locations/<location>/keyRings/<keyring>/cryptoKeys/<key>"
 ```
-`onError: [ stopAndFail | continue ]`. (Optional, default `continue`)
-`stopAndFail` will cause the build to fail if image verification fails,  `continue` will allow the build to continue even
-if verification fails.
 
-`skip-verification`. (Optional, default `true`), if true then directly return nil, if false then do the verfication
+`verify`. (Optional, default `false`), if false then directly return nil, if true then do the verification
```

Comment:
We'll have more flexibility if this is an enum rather than a bool. I like [Pod Security Admission's policy modes](https://kubernetes.io/docs/concepts/security/pod-security-admission/#pod-security-admission-labels-for-namespaces) of enforce/warn/audit - maybe this can look like:

- No policy = nothing (don't enforce)
- Policy (no mode set) = enforce
- Policy (mode = enforce/warn/audit) = use the given mode

Reply:
I think we need a mode to directly skip the whole verification? So maybe (enforce/warn/skip)?

Reply:
Oh I get it after reading the following comments. No policy applied indicates the `skip`